test-gen-plugin/src/main/java/com/dylibso/chicory/maven/JavaTestGen.java

```diff
@@ -370,10 +381,15 @@ private List<Expression> generateAssert(String varName, Command cmd) {
                 (cmd.action().args() != null)
                         ? Arrays.stream(cmd.action().args())
                                 .map(WasmValue::toArgsValue)
-                                .collect(Collectors.joining(", "))
-                        : "";
+                                .collect(Collectors.toList())
+                        : List.<String>of();
+
+        var argSize =
+                (cmd.action().args() != null)
+                        ? Arrays.stream(cmd.action().args()).mapToInt(a -> a.type().size()).sum()
+                        : 0;
 
-        var invocationMethod = ".apply(" + args + ")";
+        var invocationMethod = ".apply(" + String.join(",", args) + ")";
```

Comment:
notice, this is _not_ correct, but works for now. It should be adapted to do same thing as the lines below, but it required a little more refactoring, and I wanted to push this out quickly to unblock @mfirry